content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -38,7 +38,7 @@ No parameters are passed to the handler.
 
 This hook is called immediately before a container is terminated due to an API request or management event such as liveness probe failure, preemption, resource contention and others. A call to the preStop hook fails if the container is already in terminated or completed state.
 It is blocking, meaning it is synchronous,
-so it must complete before the call to delete the container can be sent.
+so it must complete before the signal to stop the container can be sent.
```

Comment:
What do you think about this edit:
The `PreStop` hook is blocking or synchronous. The `PreStop` hook must complete before sending the signal to stop the container.

Reply:
+1 – Strongly support @kbhawkey's rewording here.

Reply:
@kbhawkey so that I make sure I don't mangle your wording, can you use the suggested change feature to tell me exactly how you want that sentence in line here? FWIW I think "blocking or synchronous" parses as if those two are different things but I know you are trying to say they are synonyms. I could make this briefer by saying "blocking (i.e. synchronous)" if you like?